zebrad/src/components/mempool/error.rs

```diff
@@ -50,9 +47,9 @@ pub enum MempoolError {
     #[error("transaction dropped because the queue is full")]
     FullQueue,
 
+    /// The mempool is not enabled yet.
+    ///
+    /// Zebra enables the mempool once it reaches the chain tip.
```

Comment:
Nitpick: Zebra can disable the mempool if it falls behind the tip, then starts syncing lots of blocks again.

```suggestion
    /// Zebra enables the mempool when it is at the chain tip.
```